generators/client/templates/angular/src/main/webapp/app/login/login.service.ts.ejs

```diff
@@ -86,6 +86,9 @@ export class LoginService {
       // if Keycloak, uri has protocol/openid-connect/token
       if (logoutUrl.includes('/protocol')) {
         logoutUrl = logoutUrl + '?redirect_uri=' + redirectUri;
+      } else if(logoutUrl.includes('auth0.com')) {
```

Comment:
This logic isn't great because Auth0 users can have [custom domains](https://auth0.com/docs/custom-domains).

Reply:
I agree. At the moment, I don't have a better way to identify. Also, I am not very happy with the if-else blocks spreading across frontend, e2e, and backend layers to distinguish between different providers. I will create an improvement issue to handle that in a better way.